languages/reviewers.md

```diff
@@ -0,0 +1,4 @@
+# See the README.md for the definition of the roles and responsibilities
+| Languages   |   Reviewers    |     
+| :---------: | :-------------: | 
+| **Chinese** |   JieDing       |     
```

Comment:
Rather than having a single list, I suspect that having a list within the `languages/{code}` directory would be better - or a GitHub team of "zh-cn reviewers" for example.

Reply:
I re-org the languages structure. Reviewer list is now under `languages/{code}` for better self-maintain.